editor.planx.uk/src/components/Feedback/index.tsx

```diff
@@ -37,6 +36,23 @@ import {
   UserFeedback,
 } from "./types";
 
+const FeedbackPhaseBannerView: React.FC<{ 
```

Comment:
By default, the MUI Drawer retains focus on the initiating element and returns it on close ([docs](https://v5.mui.com/material-ui/react-drawer/#temporary-drawer)).

This wasn't happening here as the main `Feedback` component is a complex component which holds within in it many child components. As a result, on re-render (e.g. change of `FeedbackView`) we lose a stable ref to the button which opened the drawer and focus is lost.

By splitting this out, we are able to maintain a stable ref and the default behaviour kicks back in - resolving this issue.

A wider piece of work here would be to properly split this up and handle the state but is outside the scope of this PR.